src/main/java/entity/data_structure/DataStoreArrays.java

```diff
@@ -0,0 +1,73 @@
+package entity.data_structure;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+/**
+ * Implementation of DataStore using arrays.
+ * @param <T> item for data structure.
+ */
+public class DataStoreArrays<T> implements DataStore<T>, Iterable<T> {
+    private final ArrayList<T> data;
+
+    public DataStoreArrays() {
+        this.data = new ArrayList<>();
+    }
+
+    @Override
+    public void add(T element) {
+        data.add(element);
+    }
+
+    @Override
+    public void remove(T element) {
+        data.remove(element);
+    }
+
+    @Override
+    public T get(T element) {
+        T temp = null;
+        for (int i = 0; i < data.size(); i++) {
+            if (data.get(i).equals(element)) {
+                temp = data.get(i);
+            }
+        }
+        return temp;
+    }
+
+    /**
+     * Returns element by index.
+     * @param index integer value of index.
+     * @return value at index.
+     */
+    public T get(int index) {
+        return data.get(index);
+    }
```

Comment:
Great Idea! However, I believe it might be a little confusing to override this method. Do you think it's better to have something like getByIndex?